src/alto-ui/Tag/story.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, text } from '@storybook/addon-knobs';
+import centered from '@storybook/addon-centered';
+
+import Tag from './Tag';
+
```

Comment:
import { action } from '@storybook/addon-actions';
const onClick= action('onClick');

Reply:
more a mix of both

Reply:
👍